### PR TITLE
feat: add instance creation

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

Add instance creation

ref: MANAGER-2349 

/cc @jleveugle @marie-j

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/add/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  "pci_project_instances_new_step3_disabled_message_QUOTA_MINRAM": "L'image {{name}} requiert un minimum de {{requiredRam}} de mémoire vive (RAM) pour fonctionner correctement. L'instance sélectionnée ne dispose que de {{currentRam}}.",
+  "pci_project_instances_new_step3_quantity_info": "Toutes vos instances seront configurées de la même manière et déployées dans la même région.",
+  "pci_project_instances_new_step3_name_additional": "Le nom des instances supplémentaires sera préfixé et incrémenté à partir du terme saisi.",
+  "pci_project_instances_new_step3_network_label": "Associer au réseau privé",
+  "pci_project_instances_new_step3_network_info": "Protégez vos données grâce à un réseau privé. Il ne boostera pas seulement la connexion entre vos services, il vous permettra aussi de sécuriser les échanges de données entre les différents serveurs. <a href=\"{{url}}\" target=\"_blank\">Plus de détails sur les réseaux privés</a>.",
+
+
+  "common_cancel": "Annuler",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/add/translations/Messages_fr_FR.json]: These shouldn't be prefixed too?

File: packages/manager/modules/pci/src/projects/project/instances/add/add.html [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        data-on-focus="$ctrl.onImageFocus()"
+        data-on-submit="$ctrl.onImageChange()"
+        data-navigation="$ctrl.showImageNavigation()">
+        <pci-project-images-list
+            data-display-selected-image="$ctrl.displaySelectedImage"
+            data-flavor-type="$ctrl.model.flavorGroup.name"
+            data-region="$ctrl.model.datacenter.name"
+            data-service-name="{{:: $ctrl.projectId }}"
+            data-selected-image="$ctrl.model.image">
+            <a href="#region" class="oui-link">Pas dispo</a>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.html]: ⚠️ Must be in a translation file.

File: packages/manager/modules/pci/src/components/project/instance/ssh-keys/ssh-keys.html [hunk 3]
```diff
@@ -1,61 +1,76 @@
...
                 data-name="newSshKey_value"
                 data-rows="5"
                 data-disabled="$ctrl.loaders.isAdding"
-                data-model="$ctrl.model.key"
+                data-model="$ctrl.model.publicKey"
                 data-required="true">
             </oui-textarea>
+
+            <div class="oui-field__helper">
+                <span class="d-inline-block" scdata-translate="pci_project_instance_ssh_key_help_required"></span>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/components/project/instance/ssh-keys/ssh-keys.html]: ⚠️ unknown directive `scdata`.

File: packages/manager/modules/pci/src/projects/project/legacy/compute/infrastructure/virtualMachine/index.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import angular from 'angular';
 import '@ovh-ux/ng-translate-async-loader';
 import 'angular-translate';
 
-import add from './add';
+// import add from './add';
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/legacy/compute/infrastructure/virtualMachine/index.js]: Should be kept?

File: packages/manager/modules/pci/src/projects/project/instances/add/index.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import 'oclazyload';
+
+const moduleName = 'ovhManagerPciInstancesAddLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'oc.lazyLoad',
+  ])
+  .config(($stateProvider) => {
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/instances/add/index.js]:
```suggestion
  .config(/* @ngInject */ ($stateProvider) => {
```

File: packages/manager/modules/pci/src/projects/project/legacy/compute/infrastructure/virtualMachine/index.js (hunk 4 above)
Comment (cbourgois) [reply packages/manager/modules/pci/src/projects/project/legacy/compute/infrastructure/virtualMachine/index.js]: All this part will be removed.

**Pull request merged**